recipes/julia/rpath.patch

```diff
@@ -0,0 +1,31 @@
+--- Makefile	2017-05-06 16:34:02.000000000 +0000
++++ Makefile.new	2017-06-06 16:23:56.660979100 +0000
+@@ -414,19 +414,19 @@
+ 	# Update RPATH entries and JL_SYSTEM_IMAGE_PATH if $(private_libdir_rel) != $(build_private_libdir_rel)
+ ifneq ($(private_libdir_rel),$(build_private_libdir_rel))
+ ifeq ($(OS), Darwin)
+-	for julia in $(DESTDIR)$(bindir)/julia* ; do \
+-		install_name_tool -rpath @executable_path/$(build_private_libdir_rel) @executable_path/$(private_libdir_rel) $$julia; \
+-		install_name_tool -add_rpath @executable_path/$(build_libdir_rel) @executable_path/$(libdir_rel) $$julia; \
+-	done
++	#for julia in $(DESTDIR)$(bindir)/julia* ; do \
++	#	install_name_tool -rpath @executable_path/$(build_private_libdir_rel) @executable_path/$(private_libdir_rel) $$julia; \
++	#	install_name_tool -add_rpath @executable_path/$(build_libdir_rel) @executable_path/$(libdir_rel) $$julia; \
++	#done
+ else ifeq ($(OS), Linux)
+-	for julia in $(DESTDIR)$(bindir)/julia* ; do \
+-		patchelf --set-rpath '$$ORIGIN/$(private_libdir_rel):$$ORIGIN/$(libdir_rel)' $$julia; \
+-	done
++	#for julia in $(DESTDIR)$(bindir)/julia* ; do \
++	#	patchelf --set-rpath '$$ORIGIN/$(private_libdir_rel):$$ORIGIN/$(libdir_rel)' $$julia; \
++	#done
+ endif
+ 
+ 	# Overwrite JL_SYSTEM_IMAGE_PATH in julia library
+-	$(call stringreplace,$(DESTDIR)$(libdir)/libjulia.$(SHLIB_EXT),sys.$(SHLIB_EXT)$$,$(private_libdir_rel)/sys.$(SHLIB_EXT))
+-	$(call stringreplace,$(DESTDIR)$(libdir)/libjulia-debug.$(SHLIB_EXT),sys-debug.$(SHLIB_EXT)$$,$(private_libdir_rel)/sys-debug.$(SHLIB_EXT))
++	#$(call stringreplace,$(DESTDIR)$(libdir)/libjulia.$(SHLIB_EXT),sys.$(SHLIB_EXT)$$,$(private_libdir_rel)/sys.$(SHLIB_EXT))
```

Comment:
I don't think that's true, because the julia-0.5.2 package on bioconda uses that patch but doesn't use a wrapper script.

https://anaconda.org/bioconda/julia